LICENSES.json

```diff
@@ -1,38 +1,38 @@
 {
     "@brightlayer-ui/colors@3.0.1": {
         "licenses": "BSD-3-Clause",
-        "repository": "https://github.com/brightlayer-ui/colors",
-        "licenseUrl": "https://github.com/brightlayer-ui/colors/raw/master/LICENSE",
+        "repository": "https://github.com/etn-ccis/blui-colors",
+        "licenseUrl": "https://github.com/etn-ccis/blui-colors/raw/master/LICENSE",
         "parents": "react-native-cli-templates-demo"
     },
     "@brightlayer-ui/icons-svg@1.7.0": {
         "licenses": "BSD-3-Clause",
-        "repository": "https://github.com/brightlayer-ui/icons",
-        "licenseUrl": "https://github.com/brightlayer-ui/icons",
+        "repository": "https://github.com/etn-ccis/blui-icons",
+        "licenseUrl": "https://github.com/etn-ccis/blui-icons",
         "parents": "react-native-cli-templates-demo"
     },
     "@brightlayer-ui/react-native-auth-workflow@4.0.0-beta.0": {
         "licenses": "BSD-3-Clause",
-        "repository": "https://github.com/brightlayer-ui/react-native-workflows/tree/master/login-workflow",
-        "licenseUrl": "https://github.com/brightlayer-ui/react-native-workflows/tree/master/login-workflow/raw/master/LICENSE.md",
+        "repository": "https://github.com/etn-ccis/blui-react-native-workflows/tree/master/login-workflow",
+        "licenseUrl": "https://github.com/etn-ccis/blui-react-native-workflows/tree/master/login-workflow/raw/master/LICENSE.md",
         "parents": "react-native-cli-templates-demo"
     },
     "@brightlayer-ui/react-native-components@6.0.1": {
         "licenses": "BSD-3-Clause",
-        "repository": "https://github.com/brightlayer-ui/react-native-component-library",
-        "licenseUrl": "https://github.com/brightlayer-ui/react-native-component-library/raw/master/LICENSE",
+        "repository": "https://github.com/etn-ccis/blui-react-native-component-library",
+        "licenseUrl": "https://github.com/etn-ccis/blui-react-native-component-library/raw/master/LICENSE",
         "parents": "react-native-cli-templates-demo"
     },
     "@brightlayer-ui/react-native-themes@6.0.0": {
         "licenses": "BSD-3-Clause",
-        "repository": "https://github.com/brightlayer-ui/react-native-themes",
-        "licenseUrl": "https://github.com/brightlayer-ui/react-native-themes/raw/master/LICENSE",
+        "repository": "https://github.com/etn-ccis/blui-react-native-themes",
+        "licenseUrl": "https://github.com/etn-ccis/blui-react-native-themes/raw/master/LICENSE",
         "parents": "react-native-cli-templates-demo"
     },
     "@brightlayer-ui/react-native-vector-icons@1.3.1": {
         "licenses": "BSD-3-Clause",
-        "repository": "https://github.com/brightlayer-ui/icons",
-        "licenseUrl": "https://github.com/brightlayer-ui/icons/raw/master/LICENSE",
+        "repository": "https://github.com/etn-ccis/blui-icons",
+        "licenseUrl": "https://github.com/etn-ccis/blui-icons/raw/master/LICENSE",
```

Comment:
As rn vector icons has been moved to packages, do we need to target this license https://github.com/etn-ccis/blui-icons/blob/dev/packages/rn-vector/LICENSE ?